packages/flutter_tools/templates/plugin_ffi/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Please add comments like those I added to the other CMake files about ideally not changing this version.